### DISCUSSION TITLE
Removing HTML comments from build

### DISCUSSION THREAD

**Discussion by niconicoj** [quasarframework/quasar#16016] [General - Components / Directives / etc] [answered]
When building a quasar app there are a bunch of empty comments around the resutling html which looks like this :
```
...
<div class="q-layout q-layout--standard" tabindex="-1" style="min-height: 937px;">
  <!---->
  <!---->
  <header class="q-header q-layout__section--marginal fixed-top base-header bg-presque-noir text-white" data-v-0ca4fec7="">
 ...
```
There are no comments in any of my components. They seems to come solely from either quasar or vue.

Is there a way to remove these on the users end ?

**Comment by yusufkandemir** [comment, marked as answer]
That's how Vue manages fragments. If you create a component with an empty `template` section and use it somewhere, it will also produce `<!---->`. If you do `v-if="something"`, when `something` is `false`, you will see similar comments (`<!--v-if-->`).
Those two specific comments are QScrollObserver and QResizeObserver, which don't render anything, and that's how Vue handles components that render nothing.
